### PR TITLE
SDKS-2445_Updates to NetworkCollector 

### PR DESCRIPTION
**Pull request by george-bafaloukas-forgerock**
Updated NetworkCollector to include a more complete network connectivity test

# JIRA Ticket
[SDKS-2445](https://bugster.forgerock.org/jira/browse/SDKS-2445)

### CODE REVIEW

File: samples/quickstart/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -21,12 +14,35 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Forgerockandroidsdk"
         android:usesCleartextTraffic="true">
+
+        <activity
```
Comment (jeyanthanperiyasamy) [inline samples/quickstart/src/main/AndroidManifest.xml:18]: do we need this ResetActivity in the manifest?

File: samples/quickstart/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,18 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (jeyanthanperiyasamy) [inline samples/quickstart/src/main/AndroidManifest.xml:2]: Also add the copy right with 2023

File: samples/quickstart/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
-  ~ Copyright (c) 2021 - 2022 ForgeRock. All rights reserved.
-  ~
-  ~ This software may be modified and distributed under the terms
-  ~ of the MIT license. See the LICENSE file for details.
-  -->
-
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.forgerock.quickstart">
 
-    <!--
```
Comment (jeyanthanperiyasamy) [inline samples/quickstart/src/main/AndroidManifest.xml:12]: please add this comment back, this is for our security tools and folks

**Review by jeyanthanperiyasamy — COMMENTED** [review]
added few comments

File: forgerock-auth/src/main/java/org/forgerock/android/auth/collector/NetworkCollector.java [hunk 4]
```diff
@@ -8,8 +8,13 @@
 package org.forgerock.android.auth.collector;
 
 import android.content.Context;
+import android.net.ConnectivityManager;
```
Comment (rodrigoareis) [inline forgerock-auth/src/main/java/org/forgerock/android/auth/collector/NetworkCollector.java:11]: Update copyright

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Changes looks good. Just minor changes required already spotted by Jey

**Review by spetrov — CHANGES_REQUESTED** [review]
Please, update CHANGELOG.md as well.

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Review by jeyanthanperiyasamy — APPROVED** [review]
approved

**Pull request merged**